src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -202,39 +202,57 @@ def load_command_table(self, _):
 
     # Offer throughput
     with self.command_group('cosmosdb sql database throughput', cosmosdb_sql_sdk, client_factory=cf_sql_resources) as g:
-        g.show_command('show', 'get_sql_database_throughput')
+        g.command('show', 'get_sql_database_throughput')
+        g.command('migrate-to-autoscale', 'migrate_sql_database_to_autoscale')
+        g.command('migrate-to-manual-throughput', 'migrate_sql_database_to_manual_throughput')
```

Comment:
is it possible to make the command name shorter? e.g. only keep autoscale...

Reply:
I have merged both the cmdlets in a single command. Please have a look at the updated.